collect_app/src/main/res/layout-v17/range_widget_horizontal.xml

```diff
@@ -1,67 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you have resources with specified API level like here v17 it's not only for API 17 but for 17+. I know it's misleading but actually this file is the one we want to keep, it should be moved to general layout folder to replace the one that exists there and then this folder should be removed.

Reply:
Okay yes I understand why testDebug was not able to locate the file `range_widget_horizontal.xml`

Reply:
I think we're still missing this one unless I just can't see it?

Reply:
It's there https://github.com/opendatakit/collect/pull/3625/files#diff-68e08140c170fb774f8b1da8213e3fdc so seems fine.